src/pytest_regressions/common.py

```diff
@@ -4,6 +4,8 @@
 
 from pathlib import Path
 
+IMPORT_ERROR_MESSAGE = "library should be imported on target project"
```

Comment:
Maybe `"the library {bilname} is an optional dependency and must be installed explictly when the fixture {fixname} is used"` or some thing in those lines.

Reply:
- [ ] Update error message

Reply:
Just notice, there is no need to have a `fixname` arg because the only non-private fixture is `check`.